### DISCUSSION TITLE
List of things to do and contribute

### DISCUSSION THREAD

**Discussion by danbahrami** [rsms/inter#769] [General]
Hi 👋

I'm really interested in this project and would like to carve out some time to help this font improve. I've had a look today at getting started and, apart from a few technical hitches with the build process, the largest hurdle I have is knowing what I can help with.

The contribution guide talks in broad strokes about where help is needed, but for people with little experience of building fonts (like myself) it would be helpful to have a more detailed roadmap. This would ideally layout which characters need to be created, what needs to be improved and the priority of each task. I realise that this may be a lot of work but would definitely help the dev community contribute so I thought I'd put the suggestion out there.

P.S. I can see in robofont that certain characters are marked with various colours but I can't find any documentation that explains what each mark means. Perhaps I have missed something?

**Comment by rsms** [comment]
Great initiative!

**Comment by rsms** [comment]
[CONTRIBUTING.md](https://github.com/rsms/inter/blob/master/CONTRIBUTING.md) is a good place to start for anyone who's interested in participating.

**Comment by andreik6v** [comment]
I think the font is excellent; I like it a lot. 
As a suggestion, maybe an optimization for the web. 
I've seen it on the Beaker browser website, and the font was 500KB out of the 800KB total page size. In my opinion that is a bit much.

**Comment by Calinou** [comment]
@fasterwp I don't think font authors have much control over the file sizes. However, websites implementing Web fonts can use many tricks to optimize delivery size:

- Serve fonts in WOFF2 format in [supported browsers](https://caniuse.com/#feat=woff2), falling back to WOFF automatically. It seems Inter's WOFF2 files are about 40% smaller compared to WOFF.
- Consider subsetting fonts to include only the character sets used on the website. This is relatively complex to do, but it can be worth it.
- Add `font-display: swap;` to the `@font-face` declarations so that texts still display while the Web font is loading. This improves the user experience when loading pages over a slow connection.
